### PR TITLE
Initial enhanced link mode test [skip ci]

### LINKED ISSUE #892: Execute and automate the functional tests
Depends on #890 

As a quality engineer, I want the system test suite to be automated so that I can run the tests quickly and with little additional work allowing me to focus on other efforts.

As a developer, I want the system test suite to be automated so that it will provide testing feedback on code changes as rapidly as possible to know when there is a break in the system.

### PR DESCRIPTION
**Pull request by mhagen-vmware**
Fixes #892

### CODE REVIEW

File: tests/manual-test-cases/Group5-Functional-Tests/5-3-Enhanced-Linked-Mode.robot [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+    Set Environment Variable  TEST_URL_ARRAY  ${vc1-ip}
+    Set Environment Variable  TEST_USERNAME  Administrator@vsphere.local
+    Set Environment Variable  TEST_PASSWORD  Admin\!23
+    Set Environment Variable  BRIDGE_NETWORK  bridge
+    Set Environment Variable  EXTERNAL_NETWORK  vm-network
+    Set Environment Variable  TEST_DATASTORE  datastore1
+    Set Environment Variable  TEST_RESOURCE  cls
+    Set Environment Variable  TEST_TIMEOUT  30m
+    
+    Install VIC Appliance To Test Server  ${false}  default
```
Comment (mhagen-vmware) [inline tests/manual-test-cases/Group5-Functional-Tests/5-3-Enhanced-Linked-Mode.robot]: This is what I tested and worked for now, so I would rather stick with this until you can test the changes in your PR and they work.

**Comment by andrewtchin** [comment]
lgtm

**Comment by rajanashok** [comment]
lgtm

**Comment by mhagen-vmware** [comment]
.

File: tests/resources/Nimbus-Util.robot [hunk 2]
```diff
@@ -63,4 +63,62 @@ Create a VSAN Cluster
     Set Environment Variable  TEST_RESOURCE  cls
     Set Environment Variable  TEST_TIMEOUT  30m
     
-    Gather Host IPs
\ No newline at end of file
+    Gather Host IPs
+
+Create a Simple VC Cluster
```
Comment (matthewavery) [inline tests/resources/Nimbus-Util.robot]: This does not seem to be getting used right now. Is this for future cluster work?

Comment (mhagen-vmware) [inline tests/resources/Nimbus-Util.robot]: future :)

**Pull request merged**